src/main/java/duke/parser/Parser.java

```diff
@@ -0,0 +1,147 @@
+package duke.parser;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+
+import duke.exception.DukeException;
+import duke.exception.EmptyCommandException;
+import duke.exception.IncorrectFormatException;
+import duke.exception.InvalidCommandException;
+import duke.exception.InvalidDateTimeException;
+import duke.exception.MessageEmptyException;
+import duke.exception.TaskNotFoundException;
+import duke.storage.Storage;
+import duke.tasks.Task;
+import duke.tasks.TaskList;
+import duke.ui.Ui;
+
+
+/**
+ * Handles and interprets user commands for Duke.
+ */
+
```

Comment:
Shouldn't we not have an empty line between the documentation block and the method?